build.gradle

```diff
@@ -125,6 +125,34 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Civitaspo"
+                        email = "civitaspo@gmail.com"
+                    }
+                    developer {
+                        name = "Satoshi Akama"
+                        email = "satoshiakama@gmail.com"
+                    }
+                    developer {
+                        name = "Naotoshi Seo"
+                        email = "sonots@gmail.com"
+                    }
+                    developer {
+                        name = "Khoa Nguyen"
+                        email = "instcode@gmail.com"
+                    }
```

Comment:
@instcode Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-sftp`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-sftp` on Maven Central, from your work in #45, #46, and #47. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.